Modix.Data/Models/Promotions/PromotionCommentCreationData.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace Modix.Data.Models.Promotions
+{
+    /// <summary>
+    /// Describes an operation to create a <see cref="PromotionCommentEntity"/>.
+    /// </summary>
+    public class PromotionCommentCreationData
+    {
+        /// <summary>
+        /// See <see cref="PromotionCommentEntity.CampaignId"/>.
+        /// </summary>
+        public long CampaingId { get; set; }
```

Comment:
Small typo here - should be "CampaignId"

Reply:
Ayup